webpack.config.js

```diff
@@ -163,10 +187,6 @@ function getWebpackConfig( { extensionName = '' } = {}, argv ) {
 					use: _.compact( [
 						MiniCssExtractPlugin.loader,
 						'css-loader',
-						extensionName && {
```

Comment:
Are we intentionally removing the namespacing?

Reply:
after removing it no, but temporarily in this diff we were